README.md

```diff
@@ -20,25 +20,39 @@ file a [feature request][report-feature-request]!
 
 ![Screen capture of tool](art/screencap.gif)
 
+## Contributors
+
+This project exists thanks to all the people who contribute. [[Contribute]](CONTRIBUTING.md).
+<a href="graphs/contributors"><img src="https://opencollective.com/shapeshifter/contributors.svg?width=890" /></a>
+
+
 ## Backers
 
 Support us with a monthly donation and help us continue our activities. 
 Funds will go towards hiring new developers to work on awesome features! 
 We'll display your logo below with a link to your site. 
 🙏 [[Become a backer](https://opencollective.com/shapeshifter#backer)]
 
-<a href="https://opencollective.com/shapeshifter/backer/0/website" target="_blank"><img src="https://opencollective.com/shapeshifter/backer/0/avatar.svg"></a>
-<a href="https://opencollective.com/shapeshifter/backer/1/website" target="_blank"><img src="https://opencollective.com/shapeshifter/backer/1/avatar.svg"></a>
-<a href="https://opencollective.com/shapeshifter/backer/2/website" target="_blank"><img src="https://opencollective.com/shapeshifter/backer/2/avatar.svg"></a>
-<a href="https://opencollective.com/shapeshifter/backer/3/website" target="_blank"><img src="https://opencollective.com/shapeshifter/backer/3/avatar.svg"></a>
-<a href="https://opencollective.com/shapeshifter/backer/4/website" target="_blank"><img src="https://opencollective.com/shapeshifter/backer/4/avatar.svg"></a>
-<a href="https://opencollective.com/shapeshifter/backer/6/website" target="_blank"><img src="https://opencollective.com/shapeshifter/backer/6/avatar.svg"></a>
+Thank you to all our backers! 🙏 [[Become a backer](https://opencollective.com/shapeshifter#backer)]
+
+<a href="https://opencollective.com/shapeshifter#backers" target="_blank"><img src="https://opencollective.com/shapeshifter/backers.svg?width=890"></a>
```

Comment:
I would love to use this badge, but for some reason it is displaying one of the backers avatar's twice:

<img width="823" alt="screen shot 2017-11-06 at 8 41 07 pm" src="https://user-images.githubusercontent.com/1380747/32477432-feb938f8-c332-11e7-89b4-f4dcdd925e56.png">

Notice how the 2nd and the 6th avatar are identical. Do you know why this is?

Reply:
It's because you are both an admin and you gave money to the collective. Fix going live shortly: https://github.com/opencollective/frontend/pull/102